scripts/docker_clang_env.sh

```diff
@@ -0,0 +1,19 @@
+# Use clang instead of gcc
+export OMPI_CC=/usr/bin/clang-3.9
+export OMPI_CXX=/usr/bin/clang++-3.9
+export CC=mpicc
+export CXX=mpicxx
+
+# Add clang OpenMP runtime library to LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+
+SANITIZER=$1
```

Comment:
Why don't you use the `SANITIZE={undefined,thread}` env variable here?  It is defined in the the script `docker/jenkins/build.sh` that calls `docker_clang_env.sh`.   No need to change, I'm just trying to understand why you chose to do things this way.